travis/install_llvm.sh

```diff
@@ -0,0 +1,14 @@
+export LLVM_PATH=${HOME}/llvm/clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-ubuntu-14.04
```

Comment:
@trws and then integrate Apollo, and then write the policy raja::best_threaded_exec...